x-pack/auditbeat/module/system/packages/rpm_common.go

```diff
@@ -0,0 +1,67 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package packages
+
+import (
+	"fmt"
+	"os/exec"
+	"strconv"
+	"strings"
+	"time"
+)
+
+func rpmPackagesByExec() ([]*Package, error) {
```

Comment:
I'd give this file a `_test.go` suffix so that it's not included in Auditbeat and is only available to tests.